previewer.js

```diff
@@ -25,8 +25,13 @@ function Previewer() {
 
 	// Define function to close preview
 	this.closeWindow = function () {
-		document.getElementById("previewer").style.display = "none";
-		document.getElementById("previewer-img").src = "";
+		setTimeout(function(){
+				document.getElementById("previewer").style.display = "none";
+				document.getElementById("previewer-img").style.animation = 'pop-animation 0.5s';
+			},400);
+
+		document.getElementById("previewer-img").style.animation = 'exit-animation 0.5s';
```

Comment:
Reduce the time of the exit animation to 0.2s or so as that'd be way better from UX point of view, so the user doesn't need to wait and the animation doesn't get in the way.